source/process/sg_rest_markup.rst

```diff
@@ -86,23 +96,13 @@ Create numbered lists and procedure steps using arabic numerals for the top-leve
 Links to external web pages
 ---------------------------
 
-The quickest and easiest way to create a link in reStructuredText is to use the URL without any extra markup. For example:
-
-.. code-block:: none
-  
-  https://www.mattermost.org/manifesto/
-
-On output to HTML, the link looks like this: https://www.mattermost.org/manifesto/.
-
-You can also create a link that has link text. For example:
+Embed links into text using the following formatting, ```{text} <{url}>`_``, for example:
```

Comment:
> Embed links into text using the following formatting, ```{text} <{url}>`_``, for example:

I think doing it this way is less clear because there's so many non-alphanumeric characters involved in making a link. It's easy to think that { and } are part of the construct.